### ISSUE TITLE
[v2] mf not working in HTML header 

### ISSUE THREAD

**Issue by almogdesign** [gadicc/meteor-messageformat#127]
I have the following in my HTML it works fine in English but I'm not seeing it on the translation page and it's not translating  

```html
<header class="panel-heading">
  <h2 class="panel-title">
    {{mf 'postHarvestFungicideApplication' 'Post Harvest Fungicide Application'}}
  </h2>
</header>
```

**Comment by gadicc** [comment]
That's strange.  Is it definitely only in header or is it possible that it's not picking up anything new anymore?  This is all on dev, right?  If you do `msgfmt.forceExtract()` from `meteor shell` again, does it pick it up?

**Comment by almogdesign** [comment]
Just test I get 
'Forcing (re)-extract of all files and keys.  See results in main console log, according to your default debug level'

I20150519-13:21:19.502(3)? 2015-05-19 13:21:19.478 info:  [msgfmt:extracts]  1 string(s) added, 0 changed, and 5 marked as removed.

I'm seeing the translate at http://localhost:3000/#!translate/es 

![screen shot 2015-05-19 at 1 23 52 pm](https://cloud.githubusercontent.com/assets/169507/7700871/982a3ef4-fe2a-11e4-864d-b4a3773ccab8.png)

But as you can see not on the page 
![screen shot 2015-05-19 at 1 24 56 pm](https://cloud.githubusercontent.com/assets/169507/7700884/b8d653fe-fe2a-11e4-8ed4-65af94e20e42.png)

**Comment by gadicc** [comment]
Oh, umm, the real time updates aren't completely finished yet.  If you reload the page, any difference?

**Comment by almogdesign** [comment]
Noting with page reload, tried refreshing a couple times. I can hardcode in the meantime. We are moving towards a form builder and loading the html dynamically from the database which would already have the translation

**Comment by gadicc** [comment]
That's strange.  For debugging purposes, does `msgfmt.resetStorage();` in the client console make any difference?  Otherwise does other reactive stuff work (say, showing/changing a session variable) work in that section of your site?

**Comment by almogdesign** [comment]
msgfmt.resetStorage();
undefined

Reactive does work (not everything is reactive)

**Comment by gadicc** [comment]
That last command reloaded the page too, right?

What is the value of `msgfmt.strings.es.postHarvestFungicideApplication` in the console?  `undefined` ?

Does that key appear in http://localhost:3000/msgfmt/locale/es/0?

I'm trying to figure out what's special about this particular string :(  This isn't up on staging anywhere, right?

**Comment by almogdesign** [comment]
msgfmt.strings.es.postHarvestFungicideApplication
undefined

http://localhost:3000/msgfmt/locale/es/0 yes I get a json file 
staging - yes you should still have access, if not let me know will create username and password

**Comment by gadicc** [comment]
I still have access :)  thanks.   on staging it looks like it's working?  can you confirm?  i'm talking about that particular string.

back to http://localhost:3000/msgfmt/locale/es/0 on dev, do you see `postHarvestFungicideApplication` in there?  when you did `msgfmt.resetStorage()` did the site reload?  did you get a line in the logs like:

```
debug:  [msgfmt]  [Q] fetchLocale request for "all", url: /msgfmt/locale/all/0, sendCompiled: undefined
```

the important part is the `/0`, meaning that it's redownloading all the definitions and not updating a local cache.

we'll get to the bottom of this, just trying to get a full picture :)

**Comment by almogdesign** [comment]
Yes working on production and staging not sure why it's not working in local host

**Comment by gadicc** [comment]
Hey, I'm happy it's working on production/staging but I'm concerned about it not working no dev.  I'm going to leave this open to look at, and (non-urgently) let me know about some of the other stuff and particularly what happens the next time you add a new string.
